### ISSUE TITLE
Ability to create the first superuser with environment variables 

### ISSUE THREAD

**Issue by rutger-h** [open-zaak/open-zaak#873]
**Is your feature request related to a problem? Please describe.**
We use OpenZaak as a container in Docker Compose. We need to configure all services with script, to be able to create users and connect the services.  With OpenZaak we have to do this with Now INSERT INTO superusers SQL. Not award winning. 

**Describe the solution you'd like**
When setting up OpenZaak with Docker compose, we'd like to be able to configure OpenZaak with environment variables or other means of configuration.

**Describe alternatives you've considered**
We have a work around with inserting a user in the database.

**Comment by shazada** [comment]
We might have a solution for this, but I'm not sure how we fixed this.
I'll contact my colleagues and will report back.

**Comment by joeribekker** [comment]
See: https://open-zaak.readthedocs.io/en/latest/installation/deployment/single_server.html#environment-configuration

```bash
$ docker exec -it openzaak-0 src/manage.py createsuperuser
Gebruikersnaam: demo
E-mailadres: admin@open-zaak.gemeente.nl
Password:
Password (again):
Superuser created successfully.
```

**Comment by shazada** [comment]
Pull request is being prepared and will be done soon.

**Comment by sergei-maertens** [comment]
`src/manage.py createsuperuser` also takes CLI arguments to create this non-interactively, see the built-in help for that by using `src/manage.py createsuperuser --help`

**Comment by ThomasRitense** [comment]
@sergei-maertens I found that, but I can't set the password non-interactively. That still means I can't have a preconfigured demo environment without a manual step on the CLI.

**Comment by sergei-maertens** [comment]
Let's try and get this PR finished for 1.4.0

**Comment by sergei-maertens** [comment]
@rutger-h this is now included in Open Zaak 1.4.0, and the documentation is here: https://open-zaak.readthedocs.io/en/latest/installation/provision_superuser.html

**Comment by rutger-h** [comment]
@sergei-maertens  : thanks, super! (@ThomasRitense )
